uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/browser/BrowserChrome.java

```diff
@@ -46,7 +53,11 @@ public void handleFirstRun() {
     }
 
     @Override
-    public void navigateTo(@NonNull final String url) {
+    public void navigateTo(@NonNull final String url) throws UiObjectNotFoundException {
```

Comment:
I don't think an exception is thrown here....why do we need the throws clause?

Reply:
yes, initially I used the .click() method that's why it is causing an exception.